taxonomies/ingredients.txt

```diff
@@ -51163,2182 +51228,2054 @@ ciqual_food_code:en:53200
+description:es:Borago officinalis, la borraja, es una especie fanerógama perteneciente a la familia de las boragináceas. La forma de cocinarla es variada, pues se emplea de diferentes maneras toda la planta, desde los tallos, hasta las hojas.
+description:fr:La Bourrache ou Bourrache officinale (Borago officinalis) est une espèce de plantes annuelles de la famille des Boraginacées. Les fleurs de bourrache et les jeunes feuilles se consomment traditionnellement à l'état frais.
+description:it:La borragine (Borago officinalis L.) è una pianta erbacea annuale della famiglia delle Boraginaceae. Le foglie giovani sono variamente impiegate in cucina.
+description:nl:De bernagie (Borago officinalis), ook vaak komkommerkruid genoemd, is een gemakkelijk herkenbare, ruwharige, eenjarige plant met blauwe bloemen. Als voedsel wordt de plant vooral in het Middellandse Zeegebied gewaardeerd als onderdeel van salades en andere zomerse gerechten.
+description:pt:Borragem (Borago officinalis L.), é uma planta herbácea anual mediterrânea. Folhas e flores são utilizadas.
+# 27 in 4 @2021-09-05
 
-<en:onion
-en:pre fried onions
-da:forstegte løg
-de:vorgebratene Zwiebeln
-es:cebollas pre freidas
-fi:esipaistettu sipuli, esipaistetut sipulit
-fr:oignons préfrits, oignon préfrit
-nl:voorgefrituurde uitjes
-nn:friterad lök
-pt:cebolas pre fritas
-ru:лук жареный
-sv:forhåndsstekt løk
-# ingredient/fr:oignons-préfrits has 422 products in 4 languages @2018-12-06
+<en:vegetable
+en:cardoon, artichoke thistle, cardoons
+ca:card
+da:kardon
+de:Cardy, Kardone, Spanische Artischocke
+el:αγριοαγκινάρα
+eo:kardono
+es:cardo
+fi:kardoni
+fr:cardon
+ga:cardún
+gl:cardo
+gv:cardoon
+hu:kardi
+io:karduno
+it:cardo, cardi
+ja:カルドン
+ko:카르둔
+la:Cynara cardunculus
+nl:kardoen
+pl:kard, karczoch hiszpański
+pt:cardo, cardo-de-coalho
+ro:cardon
+ru:кардон
+sl:kardij
+sr:гардун, gardun
+sv:kardon
+th:อาร์ทิโชก
+tr:devedikeni
+yi:ארטישאק
+zh:刺苞菜蓟
+#arz:خرشف
+#ast:cardu
+#kab:taga
+#pam:cardoon
+#diq:kenger
+#scn:carduni
+#plant:wikidata:en:Q7186137
+wikidata:en:Q7223635
+ciqual_food_code:en:20054
+ciqual_food_name:en:Cardoon, raw
+ciqual_food_name:fr:Cardon, cru
+wikipedia:en:https://en.wikipedia.org/wiki/Cardoon
+#30 in 5 @2021-10-21
 
-<en:pre fried onions
-fr:oignon préfrit à l'huile de tournesol
-it:cipolla prefritta in olio di girasole
-nl:voorgebakken ui in zonnebloemolie
-sv:lök brynt i solrosolja
-# ingredient/fr:oignon-préfrit-à-l'huile-de-tournesol has 44 products in 3 languages @2018-12-06
+# en:Daikon cress
+# de:Daikonkresse
+# es:daikor cress
+# fr:pousse de daikon
+# nl:daikon cress
+# ingredient/daikon-cress has no products @2019-05-15
 
-# fr:oignons-grillés is a compound ingredient, eg:Zwiebeln gegrillt (Zwiebeln, Palmöl, Weizenmehl, Kochsalz)
 
-<en:onion
-en:Baked onion
```

Comment:
-> bg, pt

Reply:
grelhadas is already a translation of en:grilled for Portuguese, enabled ingredients processing parsing for bg